pkgs/applications/version-management/git-and-tools/subgit/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, unzip, makeWrapper, jre }:
+
+stdenv.mkDerivation {
+  name = "subgit";
```

Comment:
Add version to the name please. `"subgit-" + meta.version` or something like that.